src/main/java/com/rarchives/ripme/ripper/rippers/JabArchivesRipper.java

```diff
@@ -33,7 +33,7 @@ public String getDomain() {
 
     @Override
     public String getGID(URL url) throws MalformedURLException {
-        Pattern p = Pattern.compile("^https?://(:?www.)jabarchives.com/main/view/([a-zA-Z0-9_]+).*$");
+        Pattern p = Pattern.compile("^https?://(:?www\\.)jabarchives.com/main/view/([a-zA-Z0-9_]+).*$");
```

Comment:
You still haven't changed `:?` to `?:` to make it a non-capturing group
and it's still missing a `?` at the end to match both `http://www.jabarchives.com` and `http://jabarchives.com`
```
(:?www  .)       <- first attempt
(:?www\\.)       <- this commit
(?:www\\.)?      <- what I suggested
```

Reply:
> You still haven't changed :? to ?: to make it a non-capturing group

Thanks for catching that